ouroboros-consensus/src/Ouroboros/Consensus/Node/Run/Mock.hs

```diff
@@ -38,7 +42,8 @@ instance ( ProtocolLedgerView (SimpleBlock SimpleMockCrypto ext)
   nodeBlockMatchesHeader  = matchesSimpleHeader
   nodeBlockFetchSize      = fromIntegral . simpleBlockSize . simpleHeaderStd
   nodeIsEBB               = const Nothing
-  nodeEpochSize           = \_ _ _ -> return 100
+  nodeEpochSize           = \_ cfg _ -> return $
+    EpochSize $ 10 * maxRollbacks (protocolSecurityParam cfg)
```

Comment:
Would it be more consistent to rely on `Cardano.Chain.ProtocolConstants.kEpochSlots` here?

Reply:
The mock ledger should I think be able to make independent decisions?

Reply:
I agree it should, but do we want it to here? My intuition is to minimize differences between tests and production where possible.

Reply:
... Or should we avoid having the code too dependent on specific values..? :)

Reply:
Well right now they're both `10 * k` :)

Reply:
Fair point, let's change that ;D